<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountFrame</class>
 <widget class="QFrame" name="AccountFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>440</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(37, 37, 37);</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane { /* The tab widget frame */
    border: 0px;
}

QTabWidget::tab-bar {
    alignment: center;
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
    background-color: rgb(37, 37, 37);
	color: rgb(255, 255, 255);
    min-width: 8ex;
    padding: 2px;
	height: 30px;
	width: 180px;
	font-size: 16px;
	font-family: Trebuchet MS;
}

QTabBar::tab:hover {
	color: rgb(34, 120, 255);
}

QTabBar::tab:selected {
	color: rgb(57, 136, 255);
	border:2px solid rgb(57, 136, 255);
	border-bottom-color:rgb(57, 136, 255);
	border-left-color: transparent;
	border-top-color: transparent;
	border-right-color: transparent;
}

QTabBar::tab:first:selected {
    margin-left: 0; /* the first selected tab has nothing to overlap with on the left */
}

QTabBar::tab:last:selected {
    margin-right: 0; /* the last selected tab has nothing to overlap with on the right */
}

QTabBar::tab:only-one {
    margin: 0; /* if there is only one tab, we don't want overlapping margins */
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tabCreateAccount">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <attribute name="title">
       <string>Create account</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabGetAccount">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <attribute name="title">
       <string>Get account</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(37, 37, 37);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Get account</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditGetAccountName">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(68, 67, 67);
color: rgb(255, 255, 255);
border-radius: 4px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonGetAccount">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(95, 94, 94);
color: rgb(255, 255, 255);
border-radius: 4px;</string>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(37, 37, 37);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Input</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditGetAccountInput">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(68, 67, 67);
color: rgb(255, 255, 255);
border-radius: 4px;</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(37, 37, 37);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Output</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditGetAccountOutput">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(68, 67, 67);
color: rgb(255, 255, 255);
border-radius: 4px;</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabGetTransaction">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <attribute name="title">
       <string>Get transaction</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(37, 37, 37);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Get transaction</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditGetTransactions">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(68, 67, 67);
color: rgb(255, 255, 255);
border-radius: 4px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonGetTransactions">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(95, 94, 94);
color: rgb(255, 255, 255);
border-radius: 4px;</string>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(37, 37, 37);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Input</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditGetTransactionsInput">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(68, 67, 67);
color: rgb(255, 255, 255);
border-radius: 4px;</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(37, 37, 37);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Output</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditGetTransactionsOutput">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(68, 67, 67);
color: rgb(255, 255, 255);
border-radius: 4px;</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabGetServents">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <attribute name="title">
       <string>Get servants</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(37, 37, 37);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Get servants</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditGetServants">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(68, 67, 67);
color: rgb(255, 255, 255);
border-radius: 4px;</string>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonGetServants">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(95, 94, 94);
color: rgb(255, 255, 255);
border-radius: 4px;</string>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(37, 37, 37);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Input</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditGetServantsInput">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(68, 67, 67);
color: rgb(255, 255, 255);
border-radius: 4px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(37, 37, 37);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Output</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditGetServantsOutput">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(68, 67, 67);
color: rgb(255, 255, 255);
border-radius: 4px;</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
